hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/parser/HoodieSqlCommonAstBuilder.scala

```diff
@@ -149,144 +149,4 @@ class HoodieSqlCommonAstBuilder(session: SparkSession, delegate: ParserInterface
   private def typedVisit[T](ctx: ParseTree): T = {
     ctx.accept(this).asInstanceOf[T]
   }
-
-  /**
```

Comment:
Are all of this code copied from Spark 3.3+?  Wondering if the original PR introducing this intends to support CreateIndex for Spark 3.2 and below.  cc @huberylee